server/routes/api.js

```diff
@@ -1,9 +1,103 @@
 const routes = require('express').Router();
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+
+// Possibly move these imports into sub files when each api route has it's own js file
+const mongoose = require('mongoose');
+const Donor = require('../models/donor');
+const Family = require('../models/family');
+// wishlist = Family.readWishlist();
+mongoose.connect('mongodb://localhost/aaf_local');
```

Comment:
When we move to a production environment. Will we still be using mongoose? Also a new DB instance will be created per AAF instance, right?

Regardless, this will change to some relative path instead of localhost right?

Reply:
Yes, mongoose will carry over into production.  The URL we are using for development right now will change.  Later on, either I or Efrain can set the mongoDB URL to point to a `process.env` variable for production or a local URL for development.

Reply:
Ah, right on. So Mongoose will be the one creating each new schema(?) or data ojbect when a new family is added or a new donor registers, etc?

Reply:
Not sure what your question is getting at.  Mongoose is a tool to model the data that will be inserted into MongoDB and provides an easy API to interact with MongoDB.

Reply:
I think i am mixing up my vocabulary. I'll just keep reading. =] Its coming together though. 👍